src/Build/BuildCop/API/BuildAnalyzerConfiguration.cs

```diff
@@ -42,4 +45,37 @@ public class BuildAnalyzerConfiguration
     /// If some rules are enabled and some are not, the analyzer will be run and reports will be post-filtered.
     /// </summary>
     public bool? IsEnabled { get; internal init; }
+
+    public static BuildAnalyzerConfiguration Create(Dictionary<string, string> configDictionary)
```

Comment:
The expected dictionary comparer should be documented here.

Reply:
Documentation updated. Thank you